docker/build/installers/install_doxygen.sh

```diff
@@ -30,10 +30,9 @@ apt_get_update_and_install \
     graphviz
 
 # Build doxygen from source to reduce image size
-
-VERSION="1.8.19"
+VERSION="1.9.1"
 PKG_NAME="doxygen-${VERSION}.src.tar.gz"
-CHECKSUM="ac15d111615251ec53d3f0e54ac89c9d707538f488be57184245adef057778d3"
+CHECKSUM="da8a393b7f8b7442653475a0a4bc6eaa9ce60623440d14a82fef4e8d99da4b85"
```

Comment:
Hi @gsa88, the sha256sum you provided is not correct. Please recheck it before PR.

Reply:
@storypku I apologize for my carelessness. I changed the checksum to `67aeae1be4e1565519898f46f1f7092f1973cce8a767e93101ee0111717091d1`